elk/training/classifier.py

```diff
@@ -185,7 +190,12 @@ def fit_cv(
 
     @classmethod
     def inlp(
```

Comment:
I think we should probably just get rid of this. If we really want to explore the half-space of admissible predictors we should probably try the Conical Knowledge Decomposition thing.

Reply:
I have actually been using this occasionally, basically it just gives me a higher recall way of searching for good probes. Do you still think we should get rid of it?